moped-database/migrations/1655844474965_alter_table_public_moped_proj_phases_add_column_is_deleted/up.sql

```diff
@@ -0,0 +1,10 @@
+alter table "public"."moped_proj_phases" add column "is_deleted" boolean
```

Comment:
I thought it was interesting that the new Hasura engine version didn't capitalize the SQL keywords in auto-generated migrations. 🤷

Reply:
same, I kinda wish they'd autocapitalize.